pkg/pxc/service.go

```diff
@@ -141,6 +153,12 @@ func NewServiceProxySQLUnready(cr *api.PerconaXtraDBCluster) *corev1.Service {
 		)
 	}
 
+	if cr.CompareVersionWith("1.9.0") >= 0 {
```

Comment:
Can you copy this block to https://github.com/percona/percona-xtradb-cluster-operator/blob/main/pkg/controller/pxc/controller.go#L505 ? I know, it's not pleasant but this is the reason `upgrade-consistency` test is failing now. I created a ticket to deduplicate the logic: https://jira.percona.com/browse/K8SPXC-786

Reply:
Done! :)